### PR TITLE
Improve migration

### PR DESCRIPTION
**Pull request by ximex**

### CODE REVIEW

File: db/migrations/2021_05_23_000000_set_admin_password.php
Comment (ximex) [inline db/migrations/2021_05_23_000000_set_admin_password.php:53]: is here some migration needed?

Comment (MyIgel) [reply db/migrations/2021_05_23_000000_set_admin_password.php:53]: nope there isn't as it is only used to set the admin password on initial install as a shortcut for automated installs (for example when deploying to k8s)

File: src/Database/Migration/IMigration.php
Comment (MyIgel) [inline src/Database/Migration/IMigration.php:7]: At the moment i don't see the usecase for an additional interface here?

File: db/migrations/2021_05_23_000000_set_admin_password.php
Comment (MyIgel) [reply db/migrations/2021_05_23_000000_set_admin_password.php:53]: And also adding useless method stubs is kind of pointless?

**Comment by ximex** [comment]
I added the interface as one time the down method was missing. and i added it as on some other migrations there is also an empty down method

**Comment by MyIgel** [comment]
-> Then its better to just remove the empty methods i guess? ^^

**Comment by ximex** [comment]
Could also do that. Thought the other way is correct/wanted

**Comment by ximex** [comment]
that way ok?

**Comment by MyIgel** [comment]
Hi @ximex, this branch should be rebased on the current main branch as the other (broader) cleanup already fixed some of the changes.

**Comment by ximex** [comment]
done

**Review by MyIgel — CHANGES_REQUESTED** [review]
It would be great to have two commits (based on a rebase on main): one that formats the files (and which id can be added to the [.git-blame-ignore-revs](https://github.com/engelsystem/engelsystem/blob/main/.git-blame-ignore-revs) and one that adds the strict types and private functions.

I would do that by myself but am not allowed to (would be nice if you could check the "maintainers are allowed to change the PR code" checkbox next time :) )

**Comment by weeman1337** [comment]
> It would be great to have two commits (based on a rebase on main): one that formats the files 

I would then open a PR next week and „Further improve migration“ with another formatting :upside_down_face: Can we spec the code style changes in PHPCS, so that this does not happen?

**Comment by ximex** [comment]
waiting until this PR gets merged: https://github.com/engelsystem/engelsystem/pull/1055

**Comment by MyIgel** [comment]
@ximex the PR is merged :)

**Comment by ximex** [comment]
rebased the PR. what changes should i do now?

File: db/migrations/2020_04_07_000000_change_mysql_database_encoding_to_utf8mb4.php
Comment (MyIgel) [inline db/migrations/2020_04_07_000000_change_mysql_database_encoding_to_utf8mb4.php:36]: Imho that comment is relevant and should stay even after removing the method.

**Review by MyIgel — CHANGES_REQUESTED** [review]
Besides one small change this PR imho can be squashed and merged

File: db/migrations/2020_04_07_000000_change_mysql_database_encoding_to_utf8mb4.php
Comment (ximex) [reply db/migrations/2020_04_07_000000_change_mysql_database_encoding_to_utf8mb4.php:36]: changed

**Pull request merged**
